templates/diagnostic/ansi/errors.txt.j2

```diff
@@ -0,0 +1,31 @@
+{{ "Semantic Convention Model Validation Failures" | ansi_bold | ansi_red }}:
+
+{% for item in ctx %}
+{%- if item.error.type == "policy_violation" %}
+{%- if item.error.violation.category == "attribute_registry" %}
+{{ item.error.provenance | ansi_red}}:
+{%- if item.error.violation.id == "registry_must_be_attribute_group" %}
+  Group {{ item.error.violation.group | ansi_green }} cannot be defined in the registry.
+  not an attribute group.
+{%- elif item.error.violation.id == "nonregistry_with_id_attr" %}
+  Attribute {{ item.error.violation.attr | ansi_cyan }} cannot be defined in Group {{ item.error.violation.group | ansi_green }}
+  Attribute Group not in Attribute Registry.
+{%- elif item.error.violation.id == "registry_with_ref_attr" %}
+  Attribute {{ item.error.violation.attr | ansi_cyan }} cannot be defined in Group {{ item.error.violation.group | ansi_green }}
+  Registry cannot contain attribute references.
+{%- else %}
+UNKNOWN ATTRIBUTE REGISTRY ERROR! Please open a ticket agianst semconv with this info: 
+{{ debug(item) }}
+{%- endif %}
+{%- else %}
+{{ item.error.provenance | ansi_red }}:
```

Comment:
the violation structure is rigit and doesn't work well for collisions. I'd rather not build any infra around it (at least for now). See my other comment on free-form error desription

Reply:
We should probably push for violation to NOT be rigid, but remain as pure JSON/structured data if we go with JINJA.  I was somehow under the assumption it wasn't rigid anymore, but I didn't really dive deeply yet.

If it *is* truly rigid, then you're right, we should take your error message approach.

Reply:
I could not add arbitrary properties to it and the only way I could report all the details I needed was through abusing `violation.id`. I'm arguing in https://github.com/open-telemetry/semantic-conventions/pull/1208#discussion_r1665928442 that all we need (at least now) is to give a meaningful error message to human beings and I don't see the need for this error message to be spread between policy and jinja template.